### PR TITLE
Replace TNT with Quint

### PR DESCRIPTION
**Pull request by shonfeder**
Updates the conversion code to match the renaming of `tnt` to `quint`

This is just a mechanical renaming

### CODE REVIEW

File: tla-io/src/test/scala/at/forsyte/apalache/io/tnt/TestQuint.scala [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatestplus.junit.JUnitRunner
+
+import at.forsyte.apalache.io.quint.QuintType._
+import at.forsyte.apalache.tla.lir.{FunT1, IntT1, RecRowT1, RowT1, StrT1, TupT1, VarT1, VariantT1}
+
+@RunWith(classOf[JUnitRunner])
+class TestQuint extends AnyFunSuite {
+
+  test("TNT's named type variables are converted into numbered TlaType1 variables") {
```
Comment (bugarela) [inline tla-io/src/test/scala/at/forsyte/apalache/io/tnt/TestQuint.scala:13]: Test descriptions on this file were not updated

File: tla-io/src/test/scala/at/forsyte/apalache/io/tnt/TestQuint.scala [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      VariantT1(RowT1("t1" -> RecRowT1(RowT1(("f1" -> IntT1))), "t2" -> RecRowT1(RowT1(("f2" -> StrT1)))))
+    assert(Quint.typeToTlaType(variant) == expectedVariant)
+  }
+
+  // tictactoe.json is located in tla-io/src/test/resources/tictactoe.json
+  test("Can convert types out of tictactoe.json") {
+    val tictactoeQuintJson = scala.io.Source.fromResource("tictactoe.json").mkString
+    val tntTypes = QuintOutput.read(tictactoeQuintJson).get.types
+    // All type conversions go through
+    tntTypes.map { case (id, t) => (id, t.typ, Quint.typeToTlaType(t.typ)) }
```
Comment (bugarela) [inline tla-io/src/test/scala/at/forsyte/apalache/io/tnt/TestQuint.scala:73]:
```suggestion
    val quintTypes = QuintOutput.read(tictactoeQuintJson).get.types
    // All type conversions go through
    quintTypes.map { case (id, t) => (id, t.typ, Quint.typeToTlaType(t.typ)) }
```

File: tla-io/src/test/scala/at/forsyte/apalache/io/tnt/TestQuintIR.scala [hunk 3]
```diff
@@ -7,20 +7,20 @@ import scala.util.Success
...
 @RunWith(classOf[JUnitRunner])
-class TestTntIR extends AnyFunSuite {
+class TestQuintIR extends AnyFunSuite {
 
   // tictactoe.json is located in tla-io/src/test/resources/tictactoe.json
   test("Can load tictactoe.json") {
     val tictactoeTntJson = scala.io.Source.fromResource("tictactoe.json").mkString
-    val tntIR = TntcOutput.read(tictactoeTntJson).get
+    val tntIR = QuintOutput.read(tictactoeTntJson).get
     assert(tntIR.module.name == "tictactoe")
```
Comment (bugarela) [inline tla-io/src/test/scala/at/forsyte/apalache/io/tnt/TestQuintIR.scala:16]:
```suggestion
    val quintIR = QuintOutput.read(tictactoeTntJson).get
    assert(quintIR.module.name == "tictactoe")
```

**Review by bugarela — APPROVED** [review]
Great! Just a few comments, I guess you missed some casing variants on the test files.

File: tla-io/src/test/scala/at/forsyte/apalache/io/tnt/TestQuint.scala (hunk 1 above)
Comment (shonfeder) [reply tla-io/src/test/scala/at/forsyte/apalache/io/tnt/TestQuint.scala:13]: dang. sorry about the sloppiness :( i'll clean these all up.

**Comment by shonfeder** [comment]
Thanks for the fixes!

**Pull request merged**